data/labwc-menu-generator.1.scd

```diff
@@ -36,6 +36,10 @@ in directory-schema rather than parsing .menu and .directory files.
 *-p, --pipemenu*
 	Output in pipemenu format
 
+*-t, --terminal-prefix <command>*
+	Specify prefix for Terminal=true entries, for example `foot` or
+	`xterm -e`
+
```

Comment:
At least my version pf `scdoc` doesn't like this syntax
Error msg:
```
[4/12] Generating data/labwc-menu-gene...eson to capture output, to feed input)
FAILED: data/labwc-menu-generator.1 
/usr/bin/meson --internal exe --capture data/labwc-menu-generator.1 --feed ../data/labwc-menu-generator.1.scd -- /usr/bin/scdoc
--- stderr ---
Error at 41:3: Expected ``` and a newline to begin literal block
```

I just patched it to get it to compile:
```diff
--- labwc-menu-generator.1.scd.orig	2024-06-25 16:39:50.319997984 +1000
+++ labwc-menu-generator.1.scd	2024-06-25 16:24:34.067284083 +1000
@@ -37,8 +37,16 @@
 	Output in pipemenu format
 
 *-t, --terminal-prefix <command>*
-	Specify prefix for Terminal=true entries, for example `foot` or
-	`xterm -e`
+	Specify prefix for Terminal=true entries, for example 
+	
+	```
+	foot
+	```
+	
+	or
+	```
+	xterm -e
+	```
 
 # AUTHORS
```

Reply:
Actually ignore the newlines in above patch, needless.